R/apply_row_grp_plan.R

```diff
@@ -244,25 +245,33 @@ combine_group_cols <- function(.data, group, label, element_row_grp_loc = NULL){
             select(!!!top_grouping, !!label) %>%
             mutate(!!label := !!last(group)) %>%
             distinct()
+
+          new_row_other_cols <- lone_dat %>%
```

Comment:
We should add a comment indicating what this is doing. This is getting all the _non_ group/label cols. One question is you are also not grabbing the numeric columns. why is that? numeric cols won't be converted to blanks in the gt. So maybe being consistent and just returning NA for all columns and letting downstream handle that may be better?

Reply:
I just wanted to avoid having to conditionally convert character to "" and numeric leaving as NA. But we have to conditionally handle list columns and character columns anyway, so I'm fine being even more explicit

Reply:
I've added a comment and moved the discussion about returning NAs to a separate issue/PR